pkg/util/file_indexer.go

```diff
@@ -249,6 +249,61 @@ func RunIndexer(directory string, ignoreRules []string) (filesChanged []string,
 	return filesChanged, filesDeleted, nil
 }
 
+// DeployRunIndexer walks the given directory and returns all of the files that are found, that don't match the ignore criteria
+func DeployRunIndexer(directory string, ignoreRules []string) (files []string, err error) {
```

Comment:
Why this can't be done by already existing function - RunIndexer?
This adds a lot of duplicated code

Reply:
We did not want to modify the file used in push, since we always want to push all the files to cluster. We may be able to split the duplicate code between the functions.